### DISCUSSION TITLE
Task Assignment - SRS and Diagrams

### DISCUSSION THREAD

**Discussion by sujanajayram** [DrAlzahraniProjects/csusb_fall2024_cse6550_team2#45] [General]
Hi everyone,
The list of the tasks is attached here. please refer and respond with your name, desired task number and the task name.
Eg. your name: Task 1-Introduction (Background, Objectives, Scope).
[message.txt](https://github.com/user-attachments/files/17142904/message.txt)

**Comment by sujanajayram** [comment]
i will take the below mentioned part:
Sujana: 9 Data Flow Diagram

**Comment by Prasanth-Kandikatla** [comment]
I will take Backend Architecture Diagram

Name: Prasanth.K
Topic: 8. Backend Architecture Diagram

**Comment by farheen-akhter-code** [comment]
Name: Farheen Akhter
Topic 10 - Front end Architecture Diagram

**Comment by Jahnavi24csusb** [comment]
Name : Gummalla Jahnavi
Topic 4 Functional Requirements (Part 2): Another student documents additional functionality, such as metrics tracking, conversation history, etc.

**Comment by CSUSB-Jesse-Hernandez** [comment]
Name: Jesse Hernandez
Topic 13: Main Chatbot UI Layout: One student designs a low-fidelity wireframe of the main chatbot interaction screen, including the input box and response area.

**Comment by JasonHernandez24** [comment]
Name: Jason Hernandez
Topic 3: Functional Requirements (Part 1): One student documents the chatbot's core features, like responding to academic queries, handling user input, etc.

**Comment by ERIGELASANDHYA** [comment]
Name: Sandhya Erigela
Topic 15: Feedback and Rating Interface: One student designs how users will rate chatbot responses and interact with feedback buttons.

**Comment by GorrepatiAbhishek** [comment]
Name: Abhishek
Task: 5 Non-Functional Requirements (Performance, Usability)

**Comment by VishnuTejaG07** [comment]
Name: Gurram Vishnu Teja
Task: 14 Sidebar Design: One student creates a wireframe for the sidebar that includes user metrics, statistics, and history.

**Comment by ketha-shishir** [comment]
Name: Ketha Shishir
Task: 12 NGINX and Proxy Setup Diagram: One student diagrams how NGINX is set up and how it manages requests to the Streamlit app.

**Comment by Ajayaman2627** [comment]
Name: Ajay
Task 1
Introduction (Background, Objectives, Scope): One student writes a high-level overview of the project, including the scope and objectives.

**Comment by Siyamul33** [comment]
Name : Siyamul

Task 2 
Definitions and Acronyms: One student lists and defines all technical terms, acronyms, and important terminology.

**Comment by Arungodugu** [comment]
Name: Arun
Task 7:  System Overview Diagram: One student creates a high-level architecture diagram that shows how the different components (frontend, backend, NGINX, Docker) interact.

**Comment by shreya192-edu** [comment]
Name : Shreya
Task - 6 External Interface Requirements: One student documents the APIs and any third-party services the chatbot will interact with.

**Comment by VedakshariKanneti** [comment]
Name: Vedakshari
Task-11 Deployment Diagram: One student documents how the system is deployed and how Docker is used.
